Simplified/OPDS/NYPLOPDSAcquisition.m

```diff
@@ -106,7 +106,8 @@ @interface NYPLOPDSAcquisition ()
 @property (nonatomic, copy, nonnull) NSString *type;
 @property (nonatomic, nonnull) NSURL *hrefURL;
 @property (nonatomic, nonnull) NSArray<NYPLOPDSIndirectAcquisition *> *indirectAcquisitions;
-@property (nonatomic, nonnull) id<NYPLOPDSAcquisitionAvailability> availability;
+/// The NYPLOPDSAcquisitionAvailability property is omitted here because it's no longer a read only object.
```

Comment:
nit: while this is true, it is also the default in objective-c so I don't think it's needed.